tests/baselines/reference/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.errors.txt

```diff
@@ -0,0 +1,18 @@
+tests/cases/compiler/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.ts(8,4): error TS7006: Parameter 'arg' implicitly has an 'any' type.
+
+
+==== tests/cases/compiler/contextualSignatureInArrayElementPrefersArrayUnionMemberLibEs5.ts (1 errors) ====

```

Comment:
Could you also rename this test case? If you find it useful you could also add the example with the additional number index info from [this comment](https://github.com/microsoft/TypeScript/pull/53280#discussion_r1138684984)

Reply:
Not sure what I'd rename it to; maybe I can just leave a link and say "not yet"?

Reply:
maybe, `contextualSignatureInArrayMixedWithSignatureFromStringIndexError.ts`? 🤔